README.md

```diff
@@ -90,6 +99,20 @@ or
 
 Depending upon how you'd like to work.
 
+## Updating ens contracts
+
+We use ENS as a submodule to track the dependency on it's contracts. If you are in
+`contracts/ens-repo` directory, you can use
+
+```bash
+git submodule update --remote
+git log --pretty=oneline --reverse --ancestry-path HEAD..master
+```
+
+to first update the the ens repository and then list the commits in ens `master` branch, which are
+not in our version of `ens-repo`. _(You can use this list of commits when updating to the latest
+version of ens contracts)_
```

Comment:
Do you have to `git checkout something` in the ENS repository after this and then commit in the parent repository?
Regardless of yes/no answer I'd explicitly mention the workflow for checking out a particular commit / branch on ENS and commiting that in parent repository.

Reply:
Yes, you need. I will maybe add a section how to work with submodules as that requires a bit of knowledge.